### PR TITLE
Translation of README to PT-BR

### PR DESCRIPTION
**Pull request by gustavolbs**

### CODE REVIEW

**Comment by MikhailArkhipov** [comment]
OK, we need someone who speaks the language to review... :-)

**Comment by jakebailey** [comment]
It might be better to put any translations into distinct files like `README.pt-br.md` and just link to them at the top of the main README.

**Comment by gustavolbs** [comment]
It's what I've, except because I've used README-PT-BT.md instead of README.pt-bt.md, but I can rename It if you want.

**Comment by gustavolbs** [comment]
> It might be better to put any translations into distinct files like `README.pt-br.md` and just link to them at the top of the main README.

Ok, now they are on different files and linked.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Other Languages
```
Comment (jakebailey) [inline README.md]: H1 is the wrong thing to use here. I'd suggest doing something like:

```md
# Microsoft Python Language Server

[Português - Brasil](README.pt-br.md)

Microsoft Python Language Server implements ...
```

Comment (gustavolbs) [reply README.md]: Alright, I'll do it now

**Comment by gustavolbs** [comment]
Done

**Comment by oblitum** [comment]
Is it necessary? I'm native pt_BR speaker, but I simply don't see README translations from english as practice at all.

**Comment by MikhailArkhipov** [comment]
It's been a while and the README will be changing soon due to significant rework of the code. If you are still planning to work on this and address comments, feel free to reopen.

**Pull request closed without merging**